src/App.tsx

```diff
@@ -1,26 +1,125 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useEffect, useRef, useState } from 'react';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Error } from './components/Error';
+import { Filter } from './types/Filter';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [loading, setLoading] = useState(false);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [isActive] = useState<number>();
+  const [filter, setFilter] = useState<Filter>(Filter.All);
+  const [newTodo, setNewTodo] = useState('');
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  const setAppError = (message: string) => {
+    setErrorMessage(message);
+  };
+
+  const loadTodos = async () => {
+    setLoading(true);
+    setErrorMessage('');
+    try {
+      const todosData = await getTodos();
+
+      setTodos(todosData);
+    } catch (error) {
+      setAppError('Unable to load todos');
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    loadTodos();
+  }, []);
```

Comment:
Fix it 
<img width="854" alt="image" src="https://github.com/user-attachments/assets/bb4b56a3-7471-4082-8a09-15b55fddfdc1" />